src/main/java/com/force/i18n/grammar/LanguageDictionary.java

```diff
@@ -94,20 +94,29 @@ public LanguageDictionary(HumanLanguage language, LanguageDictionary from) {
 
     @Override
     public boolean equals(Object o) {
```

Comment:
fixed  `equals(Object)` implementation that returned inaccurate result although it's generally not used